AnnoyancesFilter/sections/mobile-app.txt

```diff
@@ -22,6 +22,7 @@
 ##head[prefix~="medium-com:"] ~ body .js-openInAppButton
 ##head[prefix~="medium-com:"] ~ body .js-postActionsBarContent > .js-readNextInteractions
 !
+m.bunjang.co.kr#?#div:matches-css(margin-bottom: 6px):has(> div:contains(앱 다운로드))
```

Comment:
Better `m.bunjang.co.kr#%#//scriptlet('set-cookie', '_bun_hide_lm', '0')`

Reply:
Done. Thank you.